packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -97,9 +104,60 @@ const fields = [
 			return item.status !== 'private';
 		},
 	},
+	{
+		id: 'sticky',
+		label: 'Sticky',
+		type: 'integer',
+		Edit: ( { field, onChange, data, hideLabelFromVision } ) => {
+			const { id, getValue } = field;
+			return (
+				<ToggleControl
+					__nextHasNoMarginBottom
+					label={ hideLabelFromVision ? '' : field.label }
+					checked={ getValue( { item: data } ) }
+					onChange={ () =>
+						onChange( { [ id ]: ! getValue( { item: data } ) } )
+					}
+				/>
+			);
+		},
+	},
 ] as Field< SamplePost >[];
 
-export const Default = ( { type }: { type: 'panel' | 'regular' } ) => {
+function toFormField(
```

Comment:
I'm a bit confused as to why we need this? Isn't this part of what DataForm does internally? The other consumer (edit-site post-edit) doesn't require it (and it shouldn't).

Reply:
This is not necessary by consumers, only if they need to use a `labelPosition` that isn't the default.
I only added this in Storybook because it makes it easier to switch between the layout & labelPosition argument types, mostly for display purposes.
If this is confusing, what I could do is only use a single field as an example for the `labelPosition` versus the entire form.

Reply:
With the latest changes (adding `labelPosition` to `form`) the `toFormField` utility is not necessary.